internal/hostport/README.md

```diff
@@ -0,0 +1,8 @@
+# Host port manager
```

Comment:
Do we usually include the CRI-O logo in the top of the README.md files?  If not, we should for context.

Reply:
added